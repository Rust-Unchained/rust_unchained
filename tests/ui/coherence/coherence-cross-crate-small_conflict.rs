//@ aux-build:trait_impl_conflict.rs

extern crate trait_impl_conflict;
use trait_impl_conflict::Foo;

trait MyLocal {}

// This creates a conflict, upstream defines Foo for isize
impl<A: MyLocal> Foo for A { } //~ ERROR E0119

impl MyLocal for isize {}

fn main() {
}
