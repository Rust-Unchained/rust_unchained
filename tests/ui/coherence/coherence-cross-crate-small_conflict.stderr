error[E0119]: conflicting implementations of trait `Foo` for type `isize`
  --> $DIR/coherence-cross-crate-small_conflict.rs:9:1
   |
LL | impl<A: MyLocal> Foo for A { }
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: conflicting implementation in crate `trait_impl_conflict`:
           - impl Foo for isize;

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0119`.
